app/models/brew.rb

```diff
@@ -86,5 +88,18 @@ def self.we_are_out_of_coffee(params)
   def self.get_limit(input)
     input.to_i != 0 ? input.to_i : 1
   end
+  
+  private
+  
+  def self.create_team_if_necessary(params)
```

Comment:
I think this method could shortened to something like a built-in find_or_create_by method. Could be worth exploring.